golang/cosmos/x/vpurse/vpurse.go

```diff
@@ -0,0 +1,99 @@
+package vpurse
+
+import (
+	"encoding/json"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/Agoric/agoric-sdk/golang/cosmos/x/swingset"
```

Comment:
Curious, no change requested: Are you using `gofmt` or `goimports`? I don’t think either takes the position that there should be three import groups (but they will preserve them). I think the three import groups are consistent with Google internal style, but neither of those tools would separate them automatically. Is this a new behavior with go modules?

Reply:
The import groups are vestige of how cosmos-sdk does them.  You're right, they weren't separated automatically.